common/flagdproxy/flagdproxy.go

```diff
@@ -120,7 +120,6 @@ func (f *FlagdProxyHandler) newFlagdProxyManifest(ownerReferences []metav1.Owner
 	replicas := int32(1)
 	args := []string{
 		"start",
-		"--metrics-port",
```

Comment:
Hey @webstradev ! I think you'll want to change this to `--management-port`, instead of removing it entirely.

We're basically using the port for more than just metrics now, so the name was misleading - but we still want the same configurability.

Reply:
In fact, I think we have a similar regression [here](https://github.com/open-feature/open-feature-operator/blob/a6fa04f590ad4ad6779ce85f4fc167b59f1b17a7/common/flagdinjector/flagdinjector.go#L394) where we also seem to be configuring the ports for the container, but failing to pass the port to the flagd startup.

Reply:
Gotcha, I'll sort that tonight

Reply:
@toddbaert I have updated the two usages ofmetrics port. 

I'm uncertain about the third place you mentioned, because if I add the --management-port flag to that start command you highlighted there then a whole bunch of tests start failing.
Happy to look into that one further if you think it is necesary though.

Reply:
OK I will investigate that and make a new issue.